common/src/main/java/com/pg85/otg/configuration/biome/BiomeConfig.java

```diff
@@ -696,6 +708,28 @@ protected void writeConfigSettings(SettingsMap writer)
              
         writer.putSetting(BiomeStandardValues.FOLIAGE_COLOR_IS_MULTIPLIER, this.foliageColorIsMultiplier,
                 "Whether the foliage color is a multiplier. See GrassColorIsMultiplier for details.");
+        
+        // Fog
+        writer.putSetting(BiomeStandardValues.FOG_COLOR, this.fogColor, "Biome fog color.");
+
+        writer.putSetting(BiomeStandardValues.FOG_DENSITY, this.fogDensity,
+                "How dense the fog is this biome is, Float value from 0.0 to 1.0.",
+                "A value of 0 produces almost no fog while a value of 1 will cover the entire screen with fog.");
```

Comment:
A value of 0 produces almost no fog while a value of 1 will cover the entire screen with fog.
A value of 0 produces almost no fog, a value of 1 covers the screen with fog.

3 words less, didn't lose any meaning :). They always tell me to make things shorter, so just a writing tip ><